Multiplatform/Localizable.xcstrings

```diff
@@ -1375,6 +1375,17 @@
         }
       }
     },
+    "search.artists" : {
+      "extractionState" : "manual",
```

Comment:
Why is this manually extracted?

Reply:
This is Xcode doing its magic, I don't know why either. I'm going to remove this.